.github/workflows/uki.yaml

```diff
@@ -0,0 +1,60 @@
+name: UKI tests
+on:
+  pull_request:
+
+concurrency:
+  group: ci-uki-${{ github.head_ref || github.ref }}-${{ github.repository }}
+  cancel-in-progress: true
+env:
+  FORCE_COLOR: 1
+jobs:
+  test-uki:
+    runs-on: kvm
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: tests/go.mod
+          cache-dependency-path: tests/go.sum
+      - name: Enable KVM group perms
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libvirt-clients libvirt-daemon-system libvirt-daemon virtinst bridge-utils qemu qemu-system-x86 qemu-system-x86 qemu-utils qemu-kvm acl udev
+          # https://github.blog/changelog/2023-02-23-hardware-accelerated-android-virtualization-on-actions-windows-and-linux-larger-hosted-runners/
+          # echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          # sudo udevadm control --reload-rules
+          # sudo udevadm trigger --name-match=kvm
+          # sudo usermod -a -G kvm,libvirt $USER
+          #
+          # TODO: Switch back to the above solution when we switch to the github runners
+          # https://askubuntu.com/a/1081326
+          sudo setfacl -m u:runner:rwx /dev/kvm
+      - name: Install earthly
+        uses: Luet-lab/luet-install-action@v1.1
+        with:
+          repository: quay.io/kairos/packages
+          packages: utils/earthly
+      - name: Build uki ISO 🔧
+        run: |
+          # Do fedora as its the smaller uki possible
+          # Use immmucore master as it has patches not released for uki
+          # Use kairos-agent main branch as it has patches not released for uki
```

Comment:
i guess this will need to be updated once those patches are merged

Reply:
yep! Although with the heavy testing, Im guessing this can be left in master for both to keep testing it